docs/1.34/tutorials/migrate-to-self-hosted.mdx

```diff
@@ -0,0 +1,100 @@
+export const meta = {
```

Comment:
@nikolasburk  I didn't see any recent tutorials to copy the format from, so this `meta` block probably needs to be tweaked!

Reply:
Hmm, the `Tutorials` tab in the sidenav actually is only a placeholder that links out to https://v1.prisma.io/tutorials/

![image](https://user-images.githubusercontent.com/4058327/152522566-a210e410-211b-4003-b5e8-98c52e0dcbec.png)

So, I'm actually not sure whether it's even possible to add this here since the sidenav item can't be extended I think...

As an easy workaround, we might consider moving this guide under FAQs as well...

Reply:
Yeah, sorry the setup is not straightforward to understand here 😅 the tutorials are actually in this private (and archived) repo: https://github.com/prisma/community-tutorial-content

Reply:
Ah, got it! Then I think FAQ's are the perfect spot 😆